coralproject/talk
# Approved comments queue

## What does this PR do?

This is the same idea as the All Comments queue, just for Approved comments.

## How do I test this PR?

- view the approved comments
- approve/reject comments. the counts should update as well as the highlighted button
